pkg/dispatchcli/cmd/install.go

```diff
@@ -218,22 +240,33 @@ func NewCmdInstall(out io.Writer, errOut io.Writer) *cobra.Command {
 	return cmd
 }
 
-func makeSSLCert(out, errOut io.Writer, configDir, namespace, domain, certName string) error {
-	subject := fmt.Sprintf("/CN=%s/O=%s", domain, domain)
-	key := path.Join(configDir, fmt.Sprintf("%s.key", domain))
-	cert := path.Join(configDir, fmt.Sprintf("%s.crt", domain))
-	var err error
-	// If cert and key exist, reuse them
-	if _, err = os.Stat(key); os.IsNotExist(err) {
-		if _, err = os.Stat(cert); os.IsNotExist(err) {
-			openssl := exec.Command(
-				"openssl", "req", "-x509", "-nodes", "-days", "365", "-newkey", "rsa:2048",
-				"-keyout", key,
-				"-out", cert,
-				"-subj", subject)
-			opensslOut, err := openssl.CombinedOutput()
-			if err != nil {
-				return errors.Wrapf(err, string(opensslOut))
+func installCert(out, errOut io.Writer, configDir, namespace, domain, certName string, tls *tlsConfig) error {
```

Comment:
So do we no longer support making SSL certs on behalf of the user?  I'd like to keep that option for super easy single step setup.